basics/Strings examples/reverse string.py

```diff
@@ -0,0 +1,11 @@
+import time
+
+x=input("Enter the string to be reversed:")
+
+z=len(x)
+rev=list(x)
+
+for i in range(z-1,-1,-1):
+    print(rev[i],end='')

```

Comment:
There are at least 3 easy ways to reverse a string or a list: `reversed()`, `.reverse()` and `[::-1]`. Why not use one of those?